### PR TITLE
Add "contact us" api endpoint

### PR DESCRIPTION
**Pull request by fullonic**
## Description

Please describe your pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. When linking to an issue, please use `refs #...` in the description of the pull request.

Partial solves issue #22 by adding `api/contact-us` endpoint.

### CODE REVIEW

File: jobs/settings.py [hunk 1]
```diff
@@ -291,3 +291,13 @@
 }
 
 NOTIFICATIONS_ASYNC_QUEUE_NAME = CELERY_LOW_QUEUE_NAME
+
+
+# SMTP Setup
+
+EMAIL_BACKEND = env("EMAIL_BACKEND", default="django.core.mail.backends.console.EmailBackend")
+EMAIL_USE_TLS = env("EMAIL_USE_TLS", default=False)
```
Comment (francescarpi) [inline jobs/settings.py:299]: I think this should be `env.bool(`. 

Could be?

File: jobsapp/utils.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    Hi, {data.get("name").title()} sent the follow mensage:
+
+    {data.get("message")}
+    """
+
+    _send_email.apply_async(
+        args=(
+            data.get("subject"),
+            message,
+            data.get("email"),
```
Comment (francescarpi) [inline jobsapp/utils.py:20]: I think email should be a list. You can try it in test. See next comment.

File: tests/jobsapp/test_endpoints.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+def api_client():
+    return APIClient()
+
+
+def test_contact_us(api_client):
+    resp = api_client.get("/api/contact-us")
+    assert resp.status_code == 405
+    data = {"name": "test", "email": "test@tester.cat", "subject": "Testing", "message": "tests"}
+    resp = api_client.post("/api/contact-us", data=data)
+    assert resp.status_code == 202
```
Comment (francescarpi) [inline tests/jobsapp/test_endpoints.py:15]: You can also check if email has been sent.

```python
from django.core import mail
assert len(mail.outbox) == 1
```
But you need celery works syncron.

```
    CELERY_TASK_ALWAYS_EAGER = True
```

You can override django's settings in `pytest_configure`  in `conftest` file.

**Review by francescarpi — COMMENTED** [review]
Nice job, mate!

File: jobs/settings.py (hunk 1 above)
Comment (fullonic) [reply jobs/settings.py]: Correct, I miss that one. Thanks :+1:

File: jobsapp/utils.py (hunk 2 above)
Comment (fullonic) [reply jobsapp/utils.py]: Email in this case is "from_email", is a singular string. I renamed it now to be more clear.

**Review by francescarpi — APPROVED** [review]

**Comment by francescarpi** [comment]
Sorry mate, but tests fail.

No matter for sonarcloud, but another checks need pass.

File: docker-compose.testing.yml [hunk 4]
```diff
@@ -17,6 +17,10 @@ services:
       SECRET_KEY: ${SECRET_KEY}
       POSTGRES_HOST: db-test
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      EMAIL_HOST: localhost
+      EMAIL_PORT: 5000
+      EMAIL_HOST_USE: testing
```
Comment (jbagot) [inline docker-compose.testing.yml:22]: There is an error here, you write EMAIL_HOST_USE

**Review by jbagot — CHANGES_REQUESTED** [review]

**Pull request merged**